huntsman/observatory.py

```diff
@@ -386,6 +401,12 @@ def take_evening_flats(self,
                 self.logger.debug('Waiting for dark-field image')
                 time.sleep(1)
 
+    def take_morning_flats(*args, **kwargs):
+        '''
+
+        '''
+        raise NotImplementedError('Morning flats not implemented yet.')
+
```

Comment:
Does this really need to be a separate method, or could we apt the existing `take_evening_flats()` method to work at either dusk or dawn? I haven't looked through it carefully but I don't expect that much would have to change, just a few signs would need to flip depending on which side of midnight it was.

Reply:
Not sure yet, I'm planning on tackling that in a separate PR though. This is just a placeholder that fits with the current style of the code.